daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/SequenceGroup.scala

```diff
@@ -70,6 +70,11 @@ abstract class SequenceTermBase(
 
   def isOrdered: Boolean
 
+  /**
+   * Overridden in sequence group ref
+   */
+  def isHidden: Boolean = false
```

Comment:
This requires a test to ensure that if the same group definition is used twice, one from a dfdl:hiddenGroupRef, the other from a regular group ref, that Daffodil recognizes this and makes two disjoint copies.